test/probes/isUnsafeSpawn.spec.js

```diff
@@ -0,0 +1,68 @@
+// Import Node.js Dependencies
+import { test } from "node:test";
+import assert from "node:assert";
```

Comment:
```suggestion
import assert from "node:assert/strict";
```
We try to either use `assert.strictEqual` or `node:assert/strict` (which make `assert.equal` to be `assert.strictEqual`)